ci/log.py

```diff
@@ -44,19 +44,29 @@ class Bcolors:
     BLUE = '\033[34m'
 
 
-def configure_default_logging(stdout_level=None):
+def configure_default_logging(
+    stdout_level=None,
+    force=False,
+):
     if not stdout_level:
         stdout_level = logging.INFO
 
     # make sure to have a clean root logger (in case setup is called multiple times)
-    logging.root.handlers.clear()
+    if force:
+        for h in logging.root.handlers[:]:
+            logging.root.removeHandler(h)
+            h.close()
 
     sh = logging.StreamHandler(stream=sys.stdout)
     sh.setLevel(stdout_level)
-    sh.setFormatter(CCFormatter(fmt='%(asctime)s [%(levelprefix)s] %(name)s: %(message)s'))
+    sh.setFormatter(CCFormatter(fmt=get_default_fmt_string()))
     logging.root.addHandler(hdlr=sh)
     logging.root.setLevel(level=stdout_level)
 
-    # both too verbose ...
+    # configure loggers
```

Comment:
I think it is better to describe why it is necessary to configure the loggers. I don't need that comment to see that the log level is set here.

Reply:
yes - admittedly, my previous comment was not very elaborated, but it did give some hints about the motivation for the cfg-change..